dump/dbus_util.cpp

```diff
@@ -138,7 +138,7 @@ bool isSystemHMCManaged(sdbusplus::bus::bus& bus)
         return false;
     }
 
-    log<level::INFO>("Util system is not HMC managed");
```

Comment:
Can you please do just the fix with this commit and other updates please do with separate commit